common/g_gametype.cpp

```diff
@@ -338,6 +338,17 @@ bool G_IsTeamGame()
 	return sv_gametype == GM_TEAMDM || sv_gametype == GM_CTF;
 }
 
+/**
+ * @brief Check if team colors are being enforced due to gametype or CVARs.
+ */
+int G_IsTeamColor(int colorpreset, bool forceteamcolor, bool forceenemycolor)
```

Comment:
The function name implies a boolean return type, rather than a color present value. Maybe this function should be changed to test whether team colors should override the user's color selection (`G_ShouldUseTeamColor`).